packages/background/src/db-schema.js

```diff
@@ -56,6 +56,16 @@ const schema = {
     builtin: {doc: "Indicating builtin network, shouldn't be deleted"},
     scanUrl: {doc: 'Url of block chain explorer'},
     selected: {doc: 'network selected by wallet'},
+    tokenList: {
+      doc: 'token list of this network',
+      ref: true,
+    },
+    token: {
+      doc: 'tokens from token list, dapp, and user of this network',
```

Comment:
token from dapp and user may be account's data. why is `token` in. network here?

Reply:
token belongs to network/address/tokenList

wallet will only delete tokens if their network got removed, not address/tokenList

Reply:
So tokens here contains token from tokenList/user adding/dapp adding, covers all the tokens displaying in ui. Is that right?

Reply:
yes, it's "token" as token in blockchain